crawler/crawler.conf

```diff
@@ -20,3 +20,22 @@
     		
     [[ python_pkg ]]		
     avoid_setns = False
+
+    [[ fprobe_container ]]
+    # parameters for fprobe timeouts
+    idle_timeout = 5
+    lifetime_timeout = 5
+
+    # fprobe must create netflow version 5
+    netflow_version = 5
+
+    # The directory where all the fprobe output data will be written to
+    output_dir = /tmp/fprobe
+
+    # The filename pattern of the files that the data collector will produce
+    # container-id, pid, and timestamp will be replaced with concrete values
+    output_filepattern = fprobe-{ifname}-{timestamp}
+
+    # The use to switch fprobe and socket-datafile collector to in order to
+    # drop root privileges
+    user = nobody
```

Comment:
I renamed these two variable names. Do you want a separate patch or also squashed in?